src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,44 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include <set>
 using namespace std;
-
+set<string> MSET;
+int CMSET(string S2)
+{
+   bool CT=false,CSET; int CF=0,F=0;
+    CSET = MSET.count(S2);
+    if(CSET)
+    {CT=true;}
+    else{
+    char Conv;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'Conv' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512320)